libs/composer/captainhook.php

```diff
@@ -0,0 +1,32 @@
+<?php declare(strict_types=1);
+/******************************************************************************
+ *
+ * This file is part of ILIAS, a powerful learning management system.
+ *
+ * ILIAS is licensed with the GPL-3.0, you should have received a copy
+ * of said license along with the source code.
+ *
+ * If this is not the case or you just want to try ILIAS, you'll find
+ * us at:
+ *      https://www.ilias.de
+ *      https://github.com/ILIAS-eLearning
+ *
+ *****************************************************************************/
+
+/**
+ * @author Lukas Scharmer <lscharmer@databay.de>
+ */
+
+if (php_sapi_name() !== 'cli') {
+    exit(1);
+}
+
+$root = realpath(__DIR__ . '/../..');
+$configFile = $root . '/captainhook.json';
+$localConfigFile = $root . '/captainhook.local.json';
```

Comment:
The problem is: This file could be located anywhere in the file system :(

Reply:
Which file: `libs/composer/captainhook.php` or `captainhook.local.json`?

Reply:
`captainhook.local.json`

Reply:
So the file should be searched from the root directory?

Reply:
No. If there is no option to retrieve the current configuration from `CaptainHook` (to check wether it is already installed, e.g. with a custom file), we could not achieve our goal to force the activation.